script/run_aeron.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/sh
+
+echo "Setting shared memory for Aeron"
+mount -t tmpfs -o remount,rw,nosuid,nodev,noexec,relatime,size=256M tmpfs /dev/shm
+APP_NAME=$(echo "desdemona" | sed s/"-"/"_"/g)
+
+exec java -cp /srv/desdemona.jar "$APP_NAME.launcher.aeron_media_driver" >>/var/log/aeron.log 2>&1
```

Comment:
I _think_ the cluster requires both of them in order to run. (But it seems like we don't need run-container.sh any more.)